app/scripts/services/http-service.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+/**
+* @ngdoc service
+* @name angularServicesApp.httpService
+* @description
+* # httpService
+* # a service that abstracts $resource for simple and agnostic CRUD network calls.
+* # any special treatment or data can be abstracted in factories.
+* # will need some work to pass $token and $serverPath for Kronos Schedule.
+* @requires $resource
```

Comment:
How are we error handling this service? What options are required and how are we communicating the minimum requirements and when some is missing?